app/src/androidTest/java/org/mozilla/fenix/syncintegration/conftest.py

```diff
@@ -58,12 +59,22 @@ def tps_addon(pytestconfig, tmpdir_factory):
 def tps_config(fxa_account, monkeypatch):
     monkeypatch.setenv('FXA_EMAIL', fxa_account.email)
     monkeypatch.setenv('FXA_PASSWORD', fxa_account.password)
-    with open ("/Users/synctesting/.jenkins/workspace/fenix/app/src/androidTest/resources/email.txt", "w") as f:
+
+    os.chdir('../../../../..')
```

Comment:
probably for readability we should use the absolute paths

Reply:
The problem here is that I need to go up in a directory. Form `~/fenix/app/src/androidTest/java/org/mozilla/fenix/syncintegration/` to `~/fenix/app/src/androidTest/` 5 levels up. I can't do that with absolute paths as far as I know. We don't know the beginning of the full path (that will depend on each one's path used) but we do know the end of the path.